src/sentry/lang/javascript/processor.py

```diff
@@ -179,7 +180,8 @@ def discover_sourcemap(result):
             # arbitrary number / best guess; most sourceMappingURLs are relative and
             # not very long.
             search_space = possibilities[-1][-300:].rstrip()
```

Comment:
If each entry in `possibilities` is binary then we're actually filtering on the last 300 bytes rather than characters now. I think decoding earlier at `parsed_body` like @BYK suggested is probably the right idea here.

Reply:
This should be ok, the start here will typically be just ascii.